src/components/menu/menu-item/menu-item.style.js

```diff
@@ -94,176 +113,189 @@ const StyledMenuItemWrapper = styled.a`
+          color: ${menuConfigVariants[menuType].color};
+
+          [data-component="icon"] {
+            color: ${menuConfigVariants[menuType].color};
+          }
+
+          ${clickToOpen &&
+          isOpen &&
+          css`
+            background-color: ${menuConfigVariants[menuType]
+              .submenuOpenedBackground};
+          `}
         }
 
         a:hover,
         button:hover {
-          background-color: var(--colorsComponentsMenuAutumnStandard600);
+          background-color: ${menuConfigVariants[menuType]
+            .submenuOpenedBackground};
           color: var(--colorsComponentsMenuYang100);
-        }
-      `}
 
-      ${showDropdownArrow &&
-      css`
-        > a,
-        > button {
-          padding-right: 32px;
+          ${!(href || clickToOpen) &&
+          css`
+            cursor: default;
+            background-color: ${menuConfigVariants[menuType]
+              .submenuItemBackground};
+            color: ${menuConfigVariants[menuType].color};
+          `}
+
+          [data-component="icon"] {
+            color: ${menuConfigVariants[menuType].color};
+          }
         }
 
-        a::before,
-        button::before {
-          display: block;
-          margin-top: -2px;
-          pointer-events: none;
-          position: absolute;
-          right: 16px;
-          top: 50%;
-          z-index: 2;
-          content: "";
-          width: 0;
-          height: 0;
-          border-top: 5px solid ${menuConfigVariants[menuType].text};
-          border-right: 4px solid transparent;
-          border-bottom: 4px solid transparent;
-          border-left: 4px solid transparent;
-        }
-      `};
-    `}
+        ${selected &&
+        css`
+          background-color: ${menuConfigVariants[menuType].submenuSelected};
 
-    ${isSearch &&
-    css`
-      padding: 2px 16px;
-    `}
+          a:focus,
+          button:focus {
+            background-color: ${menuConfigVariants[menuType].submenuSelected};
+          }
 
-    ${inFullscreenView &&
-    css`
-      ${as === "div" &&
+          a:hover,
+          button:hover {
+            background-color: var(--colorsComponentsMenuAutumnStandard600);
+            color: var(--colorsComponentsMenuYang100);
+          }
+        `}
+
+        ${showDropdownArrow &&
+        css`
+          > a,
+          > button {
+            padding-right: 32px;
+          }
+
+          a::before,
+          button::before {
+            display: block;
+            margin-top: -2px;
+            pointer-events: none;
+            position: absolute;
+            right: 16px;
+            top: 50%;
+            z-index: 2;
+            content: "";
+            width: 0;
+            height: 0;
+            border-top: 5px solid ${menuConfigVariants[menuType].text};
+            border-right: 4px solid transparent;
+            border-bottom: 4px solid transparent;
+            border-left: 4px solid transparent;
+          }
+        `};
```

Comment:
actually not me - this code existed before, Github's only showing it as new because it can't seem to understand code having been moved. But fair enough, and good catch - I'll remove it.